src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -226,10 +229,16 @@ class ContentSidebar extends PureComponent<Props, State> {
      * @param {Error} error - Error object
      * @return {void}
      */
-    errorCallback = (error: $AxiosXHR<any>): void => {
+    errorCallback = (error: Error): void => {
         /* eslint-disable no-console */
         console.error(error);
         /* eslint-enable no-console */
+
+        /* eslint-disable react/prop-types */
+        this.props.onError(error, ERROR_TYPE_CONTENT_SIDEBAR, ERROR_CODE_FETCH_FILE, {
```

Comment:
If this is passed into the `withErrorBoundary` HOC, does it need to be passed in to `onError` each time? Can we avoid passing the name in somehow?

Reply:
I assume you mean the ```type```? Yeah that can definitely be removed

Reply:
I can't totally remove it, since in `ContentPreview` we should distinguish between a preview and ContentPreview error. I will refactor slightly to add to end of the arguments and make it optional, defaulting to the prop